src/main/java/seedu/address/logic/commands/UnmarkCommand.java

```diff
@@ -0,0 +1,79 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.task.Status;
+import seedu.address.model.task.Task;
+
+/**
+ * Marks a task as incomplete.
+ */
+public class UnmarkCommand extends Command {
+    public static final String COMMAND_WORD = "unmark";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Marks the task identified by the index number used in the displayed task list as incomplete.\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_UNMARK_TASK_SUCCESS = "Task marked as incomplete: %1$s";
+    public static final String MESSAGE_UNMARK_UNMARKED_TASK = "The task is already marked as incomplete";
+
+    private final Index targetIndex;
+
+    public UnmarkCommand(Index targetIndex) {
+        this.targetIndex = targetIndex;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+        List<Task> lastShownList = model.getFilteredTaskList();
+
+        if (targetIndex.getZeroBased() >= lastShownList.size()) {
+            throw new CommandException(Messages.MESSAGE_INVALID_TASK_DISPLAYED_INDEX);
+        }
+
+        Task taskToUnmark = lastShownList.get(targetIndex.getZeroBased());
+
+        Status status = taskToUnmark.getStatus();
+        if (!status.isCompleted()) {
+            throw new CommandException(MESSAGE_UNMARK_UNMARKED_TASK);
```

Comment:
Similar to above, should we create a new Exception to handle this error?

```suggestion
            throw new TaskIncompleteException(MESSAGE_UNMARK_UNMARKED_TASK);
```